app/components/search-result.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+    numMaxChars: 400,
+    truncateDescription: true,
+    displayDescription: Ember.computed('result', 'truncateDescription', function() {
```

Comment:
I added a default value to make up for that. Caner will be adding a component of his own for the equivalent of what I'm calling `search-result`, so I'm not focusing too much on this component.